test/e2e/kubelet_perf.go

```diff
@@ -31,13 +32,18 @@ import (
 
 const (
 	// Interval to poll /stats/container on a node
-	containerStatsPollingPeriod = 10 * time.Second
+	containerStatsPollingPeriod = 3 * time.Second
```

Comment:
Because this interval has not actually being picked up by the underlying resource collector. We've been mistakenly using the 1s interval. I've decided to bump it to 3s to not generate too much traffic, while still capture the occasional spikes. In fact, we'll have to change it again once we bump the cadvisor housekeeping interval.